src/client/components/footer/imprint.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import type { FC } from 'react';
+
+const Imprint: FC = () => {
+  if (process.env.REACT_APP_EOP_IMPRINT) {
+    return (
+      <a href={process.env.REACT_APP_EOP_IMPRINT}>Imprint</a>
+    );
+  }
+  return (<></>);
```

Comment:
We can also just return `null` in this case. That would make more sense to me, than the empty fragment.

Reply:
done.